packages/design-system-react/jest.config.js

```diff
@@ -15,7 +15,13 @@ module.exports = merge(baseConfig, {
   displayName,
 
   // Add coverage ignore patterns
-  coveragePathIgnorePatterns: ['index.ts'],
+  coveragePathIgnorePatterns: [
+    'index.ts',
+    'scripts/create-component/ComponentName/',
+  ],
+
+  // Add test match ignore patterns
+  testPathIgnorePatterns: ['scripts/create-component/ComponentName/'],
```

Comment:
Ignoring component template from tests. Once the component is created it will be included in the tests and test coverage.